conda_forge_tick/auto_tick.py

```diff
@@ -336,6 +401,18 @@ def run(
         raise_commit_errors=migrator.allow_empty_commits and not rerender,
     )
 
+    if rerender:
+        # for version migrations, without check solvable or automerge, we can suppress rerender errors
+        suppress_errors = (
+            is_version_migration
+            and not context.check_solvable
+            and not context.automerge
+        )
+
+        rerender_info = _run_rerender(git_backend.cli, context, suppress_errors)
```

Comment:
We should pass suppress_errors via keyword here to make it easier on devs to change the function later.

Reply:
Never had an issue with these kinds of kinds by refactoring with PyCharm. But here we go: 953f71a171d9d2d9117782fc61ade32efb8f11a6